tests/fixtures/aes_128_gcm.rs

```diff
@@ -0,0 +1,123 @@
+[
+    // https://csrc.nist.gov/Projects/Cryptographic-Algorithm-Validation-Program/CAVP-TESTING-BLOCK-CIPHER-MODES#GCMVS
```

Comment:
Test data is categorized depending on key/nonce/plaintext/aad/tag lengths. As key/nonce/tag lengths are fixed it makes sense to include tests for plaintext/aad of different lengths.